config/initializers/airbrake.rb

```diff
@@ -1 +1,6 @@
-# This file is overwritten on deploy
+Airbrake.configure do |config|
```

Comment:
Is it worth copying the config [from the example rails app](https://github.com/alphagov/govuk-rails-app-template/blob/master/templates/initializers/airbrake.rb), which guards against the environment variable not being set?